doc/guides/west/without-west.rst

```diff
@@ -3,30 +3,67 @@
 Using Zephyr without west
 #########################
 
-There are many valid usecases to avoid using a meta-tool such as west which has
-been custom-designed for a particular project and its requirements.
-This section applies to Zephyr users who fall into one or more of these
-categories:
+This page provides information on using Zephyr without west. This is
+not recommended for beginners due to the extra effort involved. In
+particular, you will have to do work "by hand" to replace these
+features:
 
-- Already use a multi-repository management system (for example Google repo)
+- cloning the additional source code repositories used by Zephyr in
+  addition to the main zephyr repository, and keeping them up to date
+- specifying the locations of these repositories to the Zephyr build
+  system
+- flashing and debugging without understanding detailed usage of the
+  relevant host tools
 
-- Do not need additional functionality beyond compiling and linking
+Getting the Source
+------------------
 
-In order to obtain the Zephyr source code and build it without the help of
-west you will need to manually clone the repositories listed in the
-`manifest file`_ one by one, at the path indicated in it.
+In addition to downloading the zephyr source code repository itself,
+you will need to manually clone the additional projects listed in the
+:term:`west manifest` file inside that repository.
 
 .. code-block:: console
 
    mkdir zephyrproject
    cd zephyrproject
    git clone https://github.com/zephyrproject-rtos/zephyr
-   # clone additional repositories listed in the manifest file,
-   # and check out the specified revisions
+   # clone additional repositories listed in zephyr/west.yml,
+   # and check out the specified revisions as well.
 
-If you want to manage Zephyr's repositories without west but still need to
-use west's additional functionality (flashing, debugging, etc.) it is possible
-to do so by manually creating an installation:
+As you pull changes in the zephyr repository, you will also need to
+maintain those additional repositories, adding new ones as necessary
+and keeping existing ones up to date at the latest revisions.
+
+Specifying Modules
+------------------
+
+If you have west installed, the Zephyr build system will use it to set
+:ref:`ZEPHYR_MODULES <important-build-vars>`. If you don't have west
+installed and your application does not need any of these
+repositories, the build will still work.
```

Comment:
In slack there was a conversation about a developer installing west to try it out, and then later deciding not to use it.  In this case, you told the developer the only way to stop the build system from using west was to completely uninstall west (``pip3 uninstall west``).  You should mention that here as a note.

Reply:
Good idea. I've put it at the top of the page so it doesn't escape notice.